src/file-1.xoo

```diff
@@ -0,0 +1,17 @@
+// dummy line 1
+// dummy line 2
+// violate HasTag rule: xoo on line 3
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Has Tag ([xoo:HasTag](http://127.0.0.1:18571/coding_rules#rule_key=xoo%3AHasTag))

[See it in SonarQube](http://127.0.0.1:18571/project/issues?id=sample&branch=feature1&issues=AWE04bUIqzxaH43k8oEx&open=AWE04bUIqzxaH43k8oEx)

[ISSUE_KEY]: AWE04bUIqzxaH43k8oEx